components/todo/SearchBar.vue

```diff
@@ -1,18 +1,35 @@
 <template>
-  <div>
-    <input
-      type="text"
-      :placeholder="$t('search')"
-      @keyup.prevent="searchTasks"
-    />
+  <div class="todo__search">
+    <transition name="search">
+      <input
+        v-if="isInputVisible"
+        ref="searchInputRef"
+        class="search-input"
+        type="text"
+        :placeholder="$t('search')"
+        @keyup.prevent="searchTasks"
+      />
+    </transition>
```

Comment:
searching logic may need update, as in the screenshot, wrong results are shown.

Reply:
![Screenshot 2023-02-21 at 9 57 39 AM](https://user-images.githubusercontent.com/85149436/220251430-1899ad0e-33bf-49cb-b870-127318694898.png)